.github/workflows/CI.yml

```
@@ -31,6 +31,7 @@ env:
   IMAGE_NAME: aws-otel-collector
   PACKAGING_ROOT: build/packages
   ECR_REPO: aws/aws-otel-collector
+  PUBLIC_ECR_REPO: aws-otel-test/dev
```

Should this be a repo under https://gallery.ecr.aws/aws-observability/?

---

I think this is fine since it is a dev and not a prod release. People are only suppose to use this for testing.

---

Would rename the env variable to make it clear that this is for the dev repo
```suggestion
  DEV_ECR_REPO: aws-otel-test/dev
```